modules/post/windows/manage/multi_meterpreter_inject.rb

```diff
@@ -35,6 +35,11 @@ def initialize(info = {})
               stdapi_sys_process_thread_create
             ]
           }
+        },
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
```

Comment:
These injection ones should probably be `CRASH_SERVICE_DOWN`.
```suggestion
          'Stability' => [CRASH_SERVICE_DOWN],
```

Reply:
Updated